src/polyfill/api_only.js

```diff
@@ -11,60 +11,6 @@
  * @fileoverview Entry point for a polyfill that only defines the types
  * (i.e. no enforcement logic).
  */
-import {trustedTypes, TrustedTypePolicy, TrustedTypePolicyFactory} from
```

Comment:
As discussed offline, please create a separate entry point for node, and keep api_only without the enforcer object.

We could also split this in two PRs if you want, one for making enforcer support installation on a passed object, and then the polyfill entry points + demo separately.

Reply:
Done.